src/main/java/com/dataartisans/flink/dataflow/translation/FlinkStreamingPipelineTranslator.java

```diff
@@ -0,0 +1,128 @@
+package com.dataartisans.flink.dataflow.translation;
+
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+import com.google.cloud.dataflow.sdk.runners.TransformTreeNode;
+import com.google.cloud.dataflow.sdk.transforms.AppliedPTransform;
+import com.google.cloud.dataflow.sdk.transforms.PTransform;
+import com.google.cloud.dataflow.sdk.transforms.join.CoGroupByKey;
+import com.google.cloud.dataflow.sdk.values.PValue;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+
+// todo see merging with the batch translator.
```

Comment:
I suppose it makes sense to keep this separate. We could, however, put some of the code in the super class `FlinkPipelineTranslator`. Not high priority.